tests/test_crons.py

```diff
@@ -220,3 +222,62 @@ def test_capture_checkin_sdk_not_initialized():
         duration=None,
     )
     assert check_in_id == "112233"
+
+
+def test_scope_data_in_checkin(sentry_init, capture_envelopes):
+    sentry_init()
+    envelopes = capture_envelopes()
+
+    valid_keys = [
+        # Mandatory event keys
+        "type",
+        "event_id",
+        "timestamp",
+        "platform",
```

Comment:
This data is already sent in the envelope header, so we can remove it here as well.

Reply:
we will do this in a separate pr